addons/rating/migrations/11.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+COLUMN_COPIES = {
+    'rating_rating': [
+        ('res_model', None, None),
+    ],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.copy_columns(env.cr, COLUMN_COPIES)
```

Comment:
Why copying the column?

Reply:
I'm copying the column because field res_model in version 10.0 is a string with the name of  the model. In version 11.0 the field is a related to res_model_id. I copy the field to not lose the value.

Reply:
Related non stored? Then better to rename it instead of copying it

Reply:
It's stored so I have to copy field.

Reply:
OK, thanks for the info.